.gitignore

```diff
@@ -1,3 +1,5 @@
+.gitignore
```

Comment:
Are you trying to ignore .gitignore files that are autogenerated in your rails apps? Once added to git you have to force remove. Since .gitignore is already added this is safe but I'd be careful doing this.